sdk/cognitiveservices/cognitiveservices-textanalytics/review/cognitiveservices-textanalytics.api.md

```diff
@@ -0,0 +1,325 @@
+## API Report File for "@azure/cognitiveservices-textanalytics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ApiKeyCredentials } from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/identity';
+
+// @public (undocumented)
+export interface AnalyzeSentimentErrorResult extends TextAnalyticsErrorResult {
+}
+
+// @public (undocumented)
+export interface AnalyzeSentimentOptions extends TextAnalyticsOperationOptions {
+}
+
+// @public (undocumented)
+export type AnalyzeSentimentResult = AnalyzeSentimentSuccessResult | AnalyzeSentimentErrorResult;
+
+// @public (undocumented)
+export interface AnalyzeSentimentResultCollection extends Array<AnalyzeSentimentResult> {
+    modelVersion: string;
+    statistics?: RequestStatistics;
+}
+
+// @public (undocumented)
+export interface AnalyzeSentimentSuccessResult extends TextAnalyticsSuccessResult {
+    documentScores: SentimentConfidenceScorePerLabel;
+    sentences: SentenceSentiment[];
```

Comment:
>sentences [](start = 4, length = 9)

any better names than `sentences` since they are not really sentences?

Reply:
This comes from the swagger types, so I am not the best person to ask. Perhaps someone from the service team can chime in?

Reply:
If we are not happy with the name, we could do swagger transform :)

Reply:
Hmm .NET calls these `SentenceSentiments`, so how about that? https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentResult.cs